assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convert.py

```diff
@@ -1,103 +1,19 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-"""HFTransformers convert model."""
+"""PyFunc convert model."""
 
-import json
-import os
-import sys
+from azureml.model.mgmt.processors.pyfunc.convertors import PyFuncMLFLowConvertor
```

Comment:
since this is being added could we get rid of this file (across flavors) and handle under preprocess.py only?

Reply:
Updated. Thanks for the suggestion